From 8632287cf6863b580340f846ac14adf2609abdb0 Mon Sep 17 00:00:00 2001
From: Fabrice Fontaine <fontaine.fabrice@gmail.com>
Date: Sat, 2 Jun 2018 13:45:22 +0200
Subject: [PATCH] remove static

Do not force static link of tools, it breaks build with:
BR2_SHARED_LIBS=y

Patch retrieved from
https://git.buildroot.net/buildroot/tree/package/libseccomp/0001-remove-static.patch
and slighly updated to work with 2.3.3

[Upstream status: https://github.com/seccomp/libseccomp/pull/121]

Signed-off-by: Bernd Kuhls <bernd.kuhls@t-online.de>
Signed-off-by: Fabrice Fontaine <fontaine.fabrice@gmail.com>
---
 tools/Makefile.am | 2 --
 1 file changed, 2 deletions(-)

diff --git a/tools/Makefile.am b/tools/Makefile.am
index 70b4aed..ef74270 100644
--- a/tools/Makefile.am
+++ b/tools/Makefile.am
@@ -35,8 +35,6 @@ scmp_bpf_disasm_SOURCES = scmp_bpf_disasm.c bpf.h util.h
 scmp_bpf_sim_SOURCES = scmp_bpf_sim.c bpf.h util.h
 
 scmp_sys_resolver_LDADD = ../src/libseccomp.la
-scmp_sys_resolver_LDFLAGS = -static
 scmp_arch_detect_LDADD = ../src/libseccomp.la
-scmp_arch_detect_LDFLAGS = -static
 scmp_bpf_disasm_LDADD = util.la
 scmp_bpf_sim_LDADD = util.la
-- 
2.14.1

